kenware/vue-survey-app
# #set up continuous build tools

#### What does this PR do?
- set up circlecli
#### Description
- set up circleci
- write test
- setup coveralls
- setup codeclimate
#### How should this be manually tested?
- Navigate to `ch-setup-circleci-test-environment`. You will see ci badges in the readme
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#162187176](https://www.pivotaltracker.com/story/show/162187176)
